interfaces/PA-App/src/app/personal/personal.page.ts

```diff
@@ -19,11 +19,6 @@ export class PersonalPage {
 
   public isDebug: boolean = !environment.production;
 
-  public program: any;
```

Comment:
Hmm.. I thought I'd removed these lines as well..

Weird that there is no merge-conflict... 😕